src/apps/companies/apps/exports/tasks.js

```diff
@@ -35,7 +35,9 @@ const createCountry = (item) => ({
 
 const transformFullExportHistory = ({ count, results }) => ({
   count,
-  results: results.map(createCountry),
+  results: results
+    .filter((result) => result.history_type === 'insert' || 'delete')
```

Comment:
In general, it's a good practise to avoid magic strings, this could be refactored into:
```suggestion
    .filter((result) => WHITELISTED_HISTORY_TYPES.includes(result.history_type)
```

Also, the above will not work as expected, here's the working example:
```suggestion
    .filter((result) => result.history_type === 'insert' || result.history_type ===  'delete')
```